src/components/calcite-radio-group-item/calcite-radio-group-item.scss

```diff
@@ -1,101 +1,93 @@
 :host {
-  display: flex;
-  align-self: stretch;
-  cursor: pointer;
+  @apply flex self-stretch cursor-pointer;
   transition: background-color 0.1s ease-in-out, border-color 0.1s ease-in-out;
 }
 
 :host label {
-  display: flex;
+  @apply flex 
+    text-color-3 
+    m-1 
+    box-border 
+    pointer-events-none 
+    items-center;
   flex: 1;
-  color: var(--calcite-ui-text-3);
   border: 1px solid transparent;
-  margin: 4px;
-  box-sizing: border-box;
   transition: background-color 0.1s ease-in-out, border-color 0.1s ease-in-out;
-  pointer-events: none;
-  display: flex;
-  align-items: center;
 }
 
 :host([layout="horizontal"]) label {
-  justify-content: center;
+  @apply content-center;
 }
 
 // focus styles
 :host {
-  @include focus-style-base();
+  @apply focus-base;
 }
 :host(:focus) {
-  @include focus-style-inset();
+  @apply focus-inset;
 }
 
 :host([scale="s"]) label {
-  @apply text--2;
-  padding: 0.25rem 0.75rem;
+  @apply text--2 px-2;
+  padding-top: 0.125rem;
+  padding-bottom: 0.125rem;
 }
 
 :host([scale="m"]) label {
-  @apply text--1;
-  padding: 0.4rem 1rem;
+  @apply text--1 px-3;
+  padding-top: .375rem;
+  padding-bottom: .375rem;
 }
 
 :host([scale="l"]) label {
-  @apply text-1;
-  padding: 0.5rem 1.5rem;
+  @apply text-1 px-4;
+  padding-top: .625rem;
+  padding-bottom: .625rem;
 }
 
 :host(:hover) label {
-  background-color: var(--calcite-ui-foreground-2);
-  color: var(--calcite-ui-text-1);
+  @apply bg-foreground-2 text-color-1;
 }
 
 :host(:active) label {
-  background-color: var(--calcite-ui-foreground-3);
+  @apply bg-foreground-3;
 }
 
 :host([checked]) label {
-  background-color: var(--calcite-ui-brand);
-  border-color: var(--calcite-ui-brand);
-  color: var(--calcite-ui-background);
-  cursor: default;
+  @apply bg-brand border-color-brand cursor-default;
+  color: theme("backgroundColor.background");
 }
 
 :host([appearance="outline"][checked]) label {
-  background-color: var(--calcite-ui-foreground-1);
-  border-color: var(--calcite-ui-brand);
-  box-shadow: inset 0 0 0 1px var(--calcite-ui-brand);
-  color: var(--calcite-ui-brand);
+  @apply bg-foreground-1 border-color-brand;
+  box-shadow: inset 0 0 0 1px theme("backgroundColor.brand");
+  color: theme("backgroundColor.brand");
 }
 
 ::slotted(input) {
-  display: none;
+  @apply hidden;
 }
 
 // icon
 .radio-group-item-icon {
-  display: inline-flex;
-  position: relative;
-  margin: 0;
+  @apply inline-flex relative m-0;
   line-height: inherit;
   transition: $transition;
```

Comment:
Could we replace the `$transition`?

Reply:
Is there a value to use here from tailwind?

Reply:
`duration-150 ease-in-out` (https://github.com/Esri/calcite-base/blob/master/dist/_variables.scss#L2)